scripts/upload_test_results/upload_to_github.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/python3
+import glob
+import os
+import argparse
+from git import Actor
+import shutil
+import git
+from git import Repo
+import json
+import wget
+import sys
+
+def parse_ver_json(commit_name):
+    # download versions.json and find commit name
+    try:
+        ver_url = 'https://testing.zephyrproject.org/daily_tests/versions.json'
+        ver_file = wget.download(ver_url, 'versions.json')
+    except:
+        print("\nERROR: Can't download versions.json file. Script execution ended.")
+        sys.exit(1)
+
+    with open(ver_file) as json_file:
+            ver_data = json.load(json_file)
+
+            # Run for from the latest member of the list, mostlikely our commit will be at the end
+            for ver_name in reversed(ver_data):
+                try:
+                    ver_commit = ver_name.split("-")[-1]
+                    ver_commit = ver_name.split("g")[1]
+                except:
+                    print("\nCan't split that version name, skip it", ver_name)
+
+                if ver_commit == commit_name:
+                    print("\nFound existing version", ver_commit)
+                    os.remove("versions.json")
+                    return ver_name
+
+
+if __name__ == '__main__':
+
+        default_branch = 'master'
+        ver_folder_name = None
+
+        # Step 1. Setup variables below and add your credentials
+        upstream_repo = 'https://github.com/zephyrproject-rtos/test_results.git'
+        origin_repo = 'https://github.com/maksimmasalski/test_results.git'
+        branch_git_title = 'intel_daily_master_'
+        author_name = 'Maksim Masalski'
+        author_email = '<maksim.masalski@intel.com>'
+        company_name = 'Intel'
+        # Finish setup variables above
+
+        parser = argparse.ArgumentParser(description = "Script to upload Daily Test Results .xml files to the test_results repo on Github")
+        parser.add_argument('-c', "--commit_name", help = "Upload daily test results of that commit", required = True)
+        parser.add_argument('-j', '--res_dir', help="Directory with all daily test result files", required = True)
+        parser.add_argument('-g', '--git_repo_dir', help="Directory to place git repo test_results", required = True)
+        argument = parser.parse_args()
+
+        file_path = argument.res_dir
+        print("Initial test results path", file_path)
+        repo_dir = argument.git_repo_dir
+
+        if not os.path.exists(file_path):
+            print("ERROR: File path to test results is not existing")
+            sys.exit(1)
+
+        ver_folder_name = parse_ver_json(argument.commit_name)
+
+        if ver_folder_name:
+
+            try:
+                empty_repo = git.Repo.init(os.path.join(repo_dir))
+                upstream = empty_repo.create_remote('origin', upstream_repo)
```

Comment:
@nashif I thought we have standard "origin" and "upstream". How to avoid hardcode there and in comment below? Use config file where for the variable will be assigned proper string name?

Reply:
'git.Repo.init()' -> 'Repo.init()' as you already has Repo imported.

Reply:
I will make config.txt file where everyone can setup their preferences.